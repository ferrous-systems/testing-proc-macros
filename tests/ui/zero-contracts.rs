#[contracts::contracts]
fn f() {}

fn main() {}
