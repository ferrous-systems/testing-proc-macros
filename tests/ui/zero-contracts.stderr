error: no contracts were specified

         = help: add a `#[precondition]`

 --> $DIR/zero-contracts.rs:3:1
  |
3 | #[contracts]
  | ^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `contracts` (in Nightly builds, run with -Z macro-backtrace for more info)
